pyincore_viz/__init__.py

```diff
@@ -1,5 +1,7 @@
 from pyincore_viz.plotutil import PlotUtil
 from pyincore_viz.geoutil import GeoUtil
 from pyincore_viz.tabledatasetlistmap import TableDatasetListMap
+from pyincore_viz.analysis.incorehousingunitallocation import IncoreHousingunitallocation
```

Comment:
do we need to include this at __init__.py?

Reply:
we don't need it
removed~